packages/components/src/combobox-control/index.tsx

```diff
@@ -80,11 +82,11 @@ function ComboboxControl( {
 	const [ isExpanded, setIsExpanded ] = useState( false );
 	const [ inputHasFocus, setInputHasFocus ] = useState( false );
 	const [ inputValue, setInputValue ] = useState( '' );
-	const inputContainer = useRef();
+	const inputContainer = useRef< HTMLInputElement >( null );
```

Comment:
Without the `null` argument, there's a TS error here:

https://github.com/WordPress/gutenberg/blob/1a9e171d11831da9dfdb78a5bd6633206dd9567d/packages/components/src/combobox-control/index.tsx#L264

```
Pproperty) React.RefAttributes<HTMLInputElement>.ref?: React.Ref<HTMLInputElement> | undefined
Type 'MutableRefObject<HTMLInputElement | undefined>' is not assignable to type 'Ref<HTMLInputElement> | undefined'.
  Type 'MutableRefObject<HTMLInputElement | undefined>' is not assignable to type 'RefObject<HTMLInputElement>'.
    Types of property 'current' are incompatible.
      Type 'HTMLInputElement | undefined' is not assignable to type 'HTMLInputElement | null'.
        Type 'undefined' is not assignable to type 'HTMLInputElement | null'.ts(2322)
index.d.ts(134, 9): The expected type comes from property 'ref' which is declared here on type 'IntrinsicAttributes & TokenInputProps & Omit<Pick<DetailedHTMLProps<InputHTMLAttributes<HTMLInputElement>, HTMLInputElement>, "key" | keyof InputHTMLAttributes<...>>, "children" | ... 1 more ... | keyof TokenInputProps> & RefAttributes<...>'
```